src/components/Contentful/DynamicHeroClassic.vue

```diff
@@ -34,26 +34,15 @@
 							</kv-carousel>
 						</template>
 						<template v-if="isHeroImage">
-							<template v-if="buttonTo">
-								<router-link
-									:to="buttonTo"
-									v-kv-track-event="[
-										'Hero',
-										'click-hero-loancards',
-										heroMedia[0].description,
-									]"
-								>
-									<kv-contentful-img
-										v-if="heroMedia[0].url"
-										class="tw-block tw-mx-auto tw-mt-0 tw-mb-4 md:tw-mb-0"
-										:contentful-src="heroMedia[0].url"
-										:width="500"
-										fallback-format="jpg"
-										:alt="heroMedia[0].description"
-									/>
-								</router-link>
-							</template>
-							<template v-else>
+							<component
```

Comment:
This syntax avoids the duplication of `kv-contentful-img` and conditionally wraps it